tests/integration/testlib/exampleslib.py

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
Shouldn't the name be singular rather than plural? (example vs examples)?

Reply:
I changed it but honestly I think having a single file for just one method is wrong.